doc/source/installation/post-installation.rst

```diff
@@ -0,0 +1,177 @@
+.. _post-installation:
+
+===============================================================================
+Post-Installation
+===============================================================================
+
+After you have installed |Percona Server|, you may need to do the following:
+
+* Initialize the data directory. Some installation methods, such as ``apt`` or ``yum``, automatically initialize the data directory.
+* Update the ``root`` password. The data directory initialization may
```

Comment:
`may` is vague, we should say that currently only CentOS/RH installations set up the password.

Reply:
Added a comment that only CentOS/RH installations